app/src/main/res/values/styles.xml

```diff
@@ -99,7 +99,10 @@
     </style>
 
     <style name="NavigationButton" parent="Widget.AppCompat.ImageButton">
+        <item name="android:layout_width">@dimen/button_size</item>
```

Comment:
In general, be very careful about putting layout_width and height in a style – it makes it impossible to look at a layout and figure out how what the views should look like without jumping into another file. If multiple views in the file put their layout_width/height in styles, the problem only escalates.

This looks like an acceptable use case because there is so much duplication and it's pretty obvious what these buttons look like.